add KAM 80's After Dark theme

FakeDazon opened the pull request:
a theme based off KAM 80's After Dark
![image](https://user-images.githubusercontent.com/44048306/104500542-6c45cf80-5593-11eb-839d-a641043d8936.png)

Miodec commented:
Delete your public folder, its no longer used.

Miodec commented:
Also, the themes error color seems to be blending in a lot with the text. Errors should generally be really in your face, so you can spot them quickly.

FakeDazon commented:
i changed the error color to a yellow, and removed the public folder
![image](https://user-images.githubusercontent.com/44048306/104668299-afd03480-568c-11eb-8b5f-6eb972952b18.png)
